src/github.com/cjlucas/unnamedcast/worker/workers.go

```diff
@@ -315,15 +316,29 @@ type UpdateUserFeedsWorker struct {
 }
 
 func (w *UpdateUserFeedsWorker) Work(q *koda.Queue, j *koda.Job) error {
```

Comment:
Refactor workers to abstract away koda. Workers should not have to fetch their own persisted job.

IDEA: Create a single interface which handles the various calls to `koda.Job` and `PersistedJob`. Have that job be the sole argument to the worker.